0102-Binary-Tree-Level-Order-Traversal.py

```diff
@@ -0,0 +1,34 @@
+'''
+Given a binary tree, return the level order traversal of its nodes' values. (ie, from left to right, level by level).
+
+For example:
+Given binary tree [3,9,20,null,null,15,7],
+    3
+   / \
+  9  20
+    /  \
+   15   7
+return its level order traversal as:
+[
+  [3],
+  [9,20],
+  [15,7]
+]
+'''
+class Solution:
+    def levelOrder(self, root: TreeNode) -> List[List[int]]:
+        if not root:
+            return root
+        queue, res = [root], []
+        while queue:
+            cur, value = [], []
```

Comment:
Instead of cur, next_level will be  a better variable name

Reply:
okay